external-api/api-tests/v1/eStore.api-test.ts

```diff
@@ -73,9 +73,9 @@ describe('/estore', () => {
     });
   });
 
-  describe('when the input is not valid - dealIdentifier contains 100000', () => {
+  describe('when the input is not valid - dealIdentifier contains 0010000000', () => {
     it('should return a status of 200 and an empty data response', async () => {
-      const { status, body } = await post({ ...payload, dealIdentifier: '100000' }).to('/estore');
+      const { status, body } = await post({ ...payload, dealIdentifier: UKEF_ID.TEST }).to('/estore');
```

Comment:
Am I reading that input is not valid returns a 200 response? That doesn't feel right to me

Reply:
On this instance I have simply update the string value to constant value.
Since it appears to be an `estore` test I should be looking into it on my next PR.

Reply:
Gotcha, makes sense